src/workerd/jsg/jsg.h

```diff
@@ -2527,6 +2527,7 @@ class Lock {
 
   // Use to enable/disable dynamic code evaluation (via eval(), new Function(), or WebAssembly).
   void setAllowEval(bool allow);
+  void installJspi();
```

Comment:
Any reason to have this method be conditionally set if you can't even unset it?
Should this just be enabled with SetWasmJSPIEnabledCallback at isolate creation on relevant isolates?

Reply:
nvm, i get it now, ignore me